### DISCUSSION TITLE
What versions of electron is @capacitor/electron compatible with?

### DISCUSSION THREAD

**Discussion by ari62** [ionic-team/capacitor#3996] [Q&A] [answered]
Hello, I am have trouble finding documentation for @capacitor/electron . What versions of electron can I use with the latest version @capacitor/electron (https://npm.io/package/@capacitor/electron)? 
thank you

**Comment by pwespi** [comment, marked as answer]
The Capacitor Electron platform has been moved to [capacitor-community/electron](https://github.com/capacitor-community/electron). You should use the new `@capacitor-community/electron` package, which according to its README supports Electron version `^11.0.1`.
